lib/screens/place_detail/local_widgets/place_title.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:cambodia_geography/cambodia_geography.dart';
+import 'package:cambodia_geography/configs/route_config.dart';
+import 'package:cambodia_geography/constants/config_constant.dart';
+import 'package:cambodia_geography/models/places/place_model.dart';
+import 'package:cambodia_geography/models/tb_province_model.dart';
+import 'package:cambodia_geography/screens/map/map_screen.dart';
+import 'package:cambodia_geography/widgets/cg_on_tap_effect.dart';
+import 'package:flutter/material.dart';
+
+class PlaceTitle extends StatelessWidget {
+  const PlaceTitle({
+    Key? key,
+    required this.place,
+  }) : super(key: key);
+  final PlaceModel place;
+
+  @override
+  Widget build(BuildContext context) {
+    CambodiaGeography geo = CambodiaGeography.instance;
+    var colorScheme = Theme.of(context).colorScheme;
+    var textTheme = Theme.of(context).textTheme;
+    TbProvinceModel province = geo.tbProvinces.firstWhere((province) => province.code == place.provinceCode);
+    return Container(
+      color: colorScheme.surface,
+      padding: EdgeInsets.all(ConfigConstant.margin2),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Text(
+            '${place.khmer}',
+            style: textTheme.headline6?.copyWith(color: colorScheme.primary, fontWeight: FontWeight.w700),
+          ),
+          Row(
+            children: [
+              Icon(
+                Icons.location_on,
+                size: ConfigConstant.iconSize1,
+                color: colorScheme.primary,
+              ),
+              Text(
+                '${province.khmer}',
+                style: textTheme.caption,
+              ),
+            ],
+          ),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.end,
+            children: [
+              ClipRRect(
+                borderRadius: ConfigConstant.circlarRadius1,
+                child: CgOnTapEffect(
+                  onTap: () {
+                    LatLng? latLng = LatLng(10.630, 104.017);
```

Comment:
use lat lng from API, so we don't need to change it later. We still have a long time to go until use all real data. To prevent errors, you can write a function above it or disable that button where lat lng is not valid.